### ISSUE TITLE
Client not displaying certain symbols/characters in your name

### ISSUE THREAD

**Issue by Thyrador** [telegramdesktop/tdesktop#24529]
### Steps to reproduce

Change your name into something like: **ᚦᛣᚱ**

### Expected behaviour

Your name should look like this: 
![image](https://user-images.githubusercontent.com/62007301/169256313-bce6a9f2-6a01-49d3-82a1-c69d1d690dd2.png)

Web client or some third party apps work just fine.

### Actual behaviour

Your name will actually look like this: 
![image](https://user-images.githubusercontent.com/62007301/169256469-c0f656d8-b54f-4488-b2b6-b8a4ea79fcc1.png)

### Operating system

Kubuntu 22.04, Plasma 5.24.4

### Version of Telegram Desktop

3.7.3

### Installation source

Static binary from official website

**Comment by ilya-fedin** [comment]
Yeah, it's Qt, it doesn't display some characters in semibold style. Nothing could be done about that at tdesktop level.

**Comment by Thyrador** [comment]
Thanks for clearing that out. I was just hoping that it could be fixed at the application level since the characters are showing fine on text messages.

An option to switch between font styles may help aswell.

**Comment by ilya-fedin** [comment]
> since the characters are showing fine on text messages.

Well, you can reproduce that in a message by choosing bold style

> An option to switch between font styles may help aswell.

I don't think it would be implemented. Core developers aren't fine with new options usually.

**Comment by Thyrador** [comment]
> Well, you can reproduce that in a message by choosing bold style
You're right. Didn't think about that one, but it does as you say. Dang.

> I don't think it would be implemented. Core developers aren't fine with new options usually.
Let a man have his dreams :D

**Comment by ilya-fedin** [comment]
The best bet would be if someone reported that to Qt... I can't do that though as I can't even reproduce it.
